similar/main/bm.cpp

```diff
@@ -396,6 +398,7 @@ void bm_read_all(d_vclip_array &Vclip, PHYSFS_File * fp)
 }
 
 int extra_bitmap_num = 0;
+int Exit_models_loaded = 0; // only really used for D2
```

Comment:
Globals default to `0`.  Some toolchains are not clever enough to place explicitly zero-initialized variables in `.bss`.

Reply:
Is explicit initialization really ever *wrong*, though?